src/app/containers/Ad/__snapshots__/MPU.test.jsx.snap

```diff
@@ -67,7 +67,7 @@ exports[`MPU should render without gel margins at all breakpoints and gel paddin
   <section
     aria-hidden="true"
     aria-label="Advertisement 2"
-    class="c0 c1"
+    class="emotion-0 emotion-1 emotion-2"
```

Comment:
In terms of snapshot changes after the migration - only other thing I've noticed in addition to the extra <div> wrapper is this extra class being given to any element with a class attribute.